### ISSUE TITLE
Install procedure on Readme not works

### ISSUE THREAD

**Issue by snowme34** [CTrando/UCSD-Planner-Helper#50]
The current way to decide if the backend will start scraping or not is based on a 'global' environment variable. Perhaps we can make it built-in via docker files.

And therefore the command on readme does not work. It will not `export SDSCHEDULE_SCRAPE=1`. User needs to run the scripts with `.` to keep the variable.

[Reference](https://stackoverflow.com/questions/1464253/global-environment-variables-in-a-shell-script)

**Comment by CTrando** [comment]
Yeah, because the scripts are run in two different contexts, the variable being exported doesn't matter in the install script
